### DISCUSSION TITLE
How to setup SumatraPDF-settings.txt to apply to all windows users on a computer?

### DISCUSSION THREAD

**Discussion by Rokartt4209** [sumatrapdfreader/sumatrapdf#4632] [Documentation]
I want a "SumatraPDF-settings.txt" file to be common to all windows users on a computer.  (Think RDP sessions on a server.) I do not want to preserve any user specific settings.  Each time Sumatra is opened by a user it should just be like they are opening it for the first time.  Sumatra will always be opened using ShellExecute with a command line to tell it the path and filename to open.  It will be configured to NOT remember the previously opened files.

**Comment by GitHubRulesOK** [comment]
Consider using restrictions.ini so the settings files do not update

However SumatraPDF is designed to be primarily per user rather than "secured against user tampering" so you need to read the options in https://www.sumatrapdfreader.org/docs/Configure-for-restricted-use

As an admin user I can potentially bypass some restrictions so use after thorough testing as if somebody else had restricted your own personal usage.

**Reply by Rokartt4209** [reply]
It looks like sumatrapdfrestrict.ini located in same folder as the .exe will be perfect and it gives me more control than I ever expected!  Thank you.
